.circleci/config.yml

```diff
@@ -59,29 +59,6 @@ commands:
           command: yarn publish << parameters.release_type >> --yes
 
 jobs:
-  test:
-    executor:
-      name: default
-    steps:
-      - prepare
-      - run: |
-          echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > ~/.npmrc
```

Comment:
please keep the npm token part in circleci until we move release workflow to github actions